### ISSUE TITLE
Separator chars (':' and '=') inside the key are not supported

### ISSUE THREAD

**Issue by luca1388** [ryanpcmcquen/propertiesToJSON#2]
For reasons out of my control i needed to use some separator characters (a colon) inside the keys of a properties file,  something as:
```
key.with.aspect.ratio.16:9 = The aspect ratio is 16:9
```
I know it is quite uncommon but seems properties file standard allows the usage of special characters inside the key by escaping them:
```
# Example with ':'
key.with.aspect.ratio.16\:9 = The aspect ratio is 16:9
# Example with '='
key.for.equation.e\=mc2 = Mass–energy equivalence
```
The current output of the tool running on the previous lines is the following:
```
{
  'key.with.aspect.ratio.16\\': '9 : The aspect ratio is 16:9',
  'key.for.equation.e\\': 'mc2 = Mass–energy equivalence'
}
```
As you can see keys are broken from the first separator char even if escaped. The expected result would be instead:
```
{
  'key.with.aspect.ratio.16:9': 'The aspect ratio is 16:9',
  'key.for.equation.e=mc2': 'Mass–energy equivalence'
}
```
It would be great to add support for this. If you like it I can try to create a pull request with a candidate fix.

**Comment by ryanpcmcquen** [comment]
Please do @luca1388!

**Comment by ryanpcmcquen** [comment]
Thanks @luca1388! Fixed in 0.2.0.
